### PR TITLE
Add content in job.md

### PR DESCRIPTION
**Pull request by ydFu**
Add description to make it easy to read.
* Distinguish between 'command'and 'output'.
* Add Code blocks to the Markdown spec.

### CODE REVIEW

File: content/en/docs/concepts/workloads/controllers/job.md [hunk 1]
```diff
@@ -38,7 +38,8 @@ You can run the example with this command:
 ```shell
 kubectl apply -f https://kubernetes.io/examples/controllers/job.yaml
 ```
-```
+The output is similar to this:
+```shell
```
Comment (sftim) [inline content/en/docs/concepts/workloads/controllers/job.md:42]: This line is incorrect @ydFu
It should be: <tt>\`\`\`</tt>

File: content/en/docs/concepts/workloads/controllers/job.md [hunk 2]
```diff
@@ -47,7 +48,8 @@ Check on the status of the Job with `kubectl`:
 ```shell
 kubectl describe jobs/pi
 ```
-```
+The output is similar to this:
+```shell
```
Comment (sftim) [inline content/en/docs/concepts/workloads/controllers/job.md:52]: This line is incorrect
It should be: <tt>\`\`\`</tt>

File: content/en/docs/concepts/workloads/controllers/job.md [hunk 3]
```diff
@@ -91,7 +93,8 @@ To list all the Pods that belong to a Job in a machine readable form, you can us
 pods=$(kubectl get pods --selector=job-name=pi --output=jsonpath='{.items[*].metadata.name}')
 echo $pods
 ```
-```
+The output is similar to this:
+```shell
```
Comment (sftim) [inline content/en/docs/concepts/workloads/controllers/job.md:97]: This line is incorrect
It should be: <tt>\`\`\`</tt>

**Review by sftim — CHANGES_REQUESTED** [review]
Thanks @ydFu 

With a couple of corrections, this will be good to merge.

File: content/en/docs/concepts/workloads/controllers/job.md (hunk 1 above)
Comment (ydFu) [reply content/en/docs/concepts/workloads/controllers/job.md]: Thanks for your guidance.

File: content/en/docs/concepts/workloads/controllers/job.md (hunk 2 above)
Comment (ydFu) [reply content/en/docs/concepts/workloads/controllers/job.md]: Thanks for your guidance.

File: content/en/docs/concepts/workloads/controllers/job.md (hunk 3 above)
Comment (ydFu) [reply content/en/docs/concepts/workloads/controllers/job.md]: Thanks for your guidance.

**Comment by ydFu** [comment]
@sftim 
Thanks for your guidance in this code 
I finished adjustment,please review this PR.

**Review by sftim — COMMENTED** [review]
/kind cleanup
/lgtm
/approve

**Pull request merged**